docs/src/main/paradox/getting-started/running-cluster.md

```diff
@@ -20,36 +20,36 @@ Add the `ShoppingCartClusterApp` to your project:
 
 Scala
 :  @@snip [ShoppingCartClusterApp.scala](/examples/src/test/scala/docs/guide/ShoppingCartClusterApp.scala) { #guideClusterSetup }
+
+Java
+:  @@snip [ShoppingCartClusterApp.java](/examples/src/test/java/jdocs/guide/ShoppingCartClusterApp.java) { #guideClusterSetup }
     
 Before running the app we must first run the `EventGeneratorApp` in `cluster` mode in order to generate new shopping cart events for multiple tags, instead of just one.
 Shopping cart events are tagged in a similar way to the sharded entities themselves.
 Given a sequence of tags from `0..n` a hash is generated using the sharding entity key, the shopping cart id.
 The hash is modded `%` by the number of tags in the sequence to choose a tag from the sequence.
 See the @ref:[Tagging Events in EventSourcedBehavior](../running.md#tagging-events-in-eventsourcedbehavior) section of the documentation for an example of how events can be tagged with Akka Persistence, or look at the source of the `EventGeneratorApp` in the previous @ref:[Running the Projection](running.md) section.
 
-Run the same `EventGeneratorApp` from the previous @ref:[Running the Projection](running.md) section, with an additional argument `cluster`:
+The same `EventGeneratorApp` from the previous @ref:[Running the Projection](running.md) section can be used to generate events for this app with an additional argument `cluster`.
+Run the app with @scala[sbt: `sbt "runMain docs.guide.EventGeneratorApp cluster"`]@java[maven: `mvn compile exec:java -Dexec.mainClass="jdocs.guide.EventGeneratorApp" -Dexec.args="cluster"`].
```

Comment:
why not tabs for sbt and maven?

Reply:
@patriknw I couldn't figure out how to do tabs for inline snippets.  Can you point me to an example?

Reply:
Search for `## Dependency` in https://raw.githubusercontent.com/lightbend/akka-enhancements/master/docs/src/main/paradox/persistence-dc/index.md

Reply:
Perfect